### PR TITLE
Make it possible to disable cross-sell on cart page

### PR DESCRIPTION
**Pull request by arnoudhgz**
### Description
To be able to select whether to have the cross-sell active on the cart page an admin configuration is created under Stores > Configuration > Sales > Checkout > Shopping Cart.

The default for this setting is set to Yes (Show cross-sell items). This is for backwards compatibility and not have webshop owners accidentally not showing these items.

Because this is now sorted as the 3rd configuration in this section/group the extra configurations for the grouped product image and the configurable product image are now also increased.

This feature will make it possible to have this cross-sell turned off during special sales, A/B testing or in a specific store view.

### Manual testing scenarios
1. Set up a cross-sell
2. Add that product that has the cross-sell product configured to your cart
3. Go to the shopping cart
4. See that cross-sell is shown
5. Change the new setting 'Show Crosssell Items in the Shopping Cart' to 'No' _(setting is under Stores > Configuration > Sales > Checkout > Shopping Cart)_
6. Refresh affected caches
7. Go back to the shopping cart
8. See that cross-sell is not shown

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=16569) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @arnoudhgz. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

**Comment by arnoudhgz** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @arnoudhgz. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @arnoudhgz, here is your [new Magento instance](https://pr-16569.engcom.dev.magento.com).
Admin access: [https://pr-16569.engcom.dev.magento.com/admin](https://pr-16569.engcom.dev.magento.com/admin)
Login: `admin` Password: `123123q`
Please make sure you are PR **author** or **assignee** to access the instance.

File: app/code/Magento/Checkout/etc/adminhtml/system.xml [hunk 1]
```diff
@@ -41,6 +41,10 @@
                 <field id="number_items_to_display_pager" translate="label" type="text" sortOrder="2" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
                     <label>Number of Items to Display Pager</label>
                 </field>
+                <field id="crosssell_enabled" translate="label" type="select" sortOrder="3" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                    <label>Show Crosssell Items in the Shopping Cart</label>
```
Comment (VladimirZaets) [inline app/code/Magento/Checkout/etc/adminhtml/system.xml]: Please add the translation to CSV for the new phrase.

Comment (arnoudhgz) [reply app/code/Magento/Checkout/etc/adminhtml/system.xml]: @VladimirZaets  Thank you for your feedback, I have added the translation.

**Comment by magento-engcom-team** [comment]
Hi @arnoudhgz. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.0. 
Please check the release notes for final confirmation.

**Pull request merged**